quoridor-app/environment.js

```diff
@@ -2,7 +2,7 @@ import { Constants } from "expo";
 
 const ENV = {
   dev: {
-    API_BASE_URL: "[your.dev.api.here]"
+    API_BASE_URL: "http://192.168.86.56:8383"
```

Comment:
IMO, this ip is specific to your local environment, isn't it ? You should create an `environment.js.dist` with fake but explicit values, exlude this `environment.js` from git and add something about that in the `README` file.

Reply:
If you follow Alexis advice about the `.dist`. You can add `cp -n environment.js.dist environment.js` in your `make install`

Reply:
environment.js is imported in gameApi so  cannot do in that way

Reply:
The idea is to exlude the `environment.js` file from the git versioning in order not to risk exposing *secret* parameters But as Maxime explains, you have to copy an `environment.dist` as `environment.js` during the installation phase, and explain to your collaborators how to set the *adhoc* configuration in it in the `README` file.